Compilation warning

```c
qterminal/src/bookmarkswidget.cpp:184:21: warning: 7 enumeration values not handled in switch: 'NoToken',
      'StartDocument', 'Characters'... [-Wswitch]
            switch (xml.tokenType())
                    ^
1 warning generated.
```

--- a/src/bookmarkswidget.cpp
+++ b/src/bookmarkswidget.cpp
@@ -221,10 +221,10 @@ public:
                          << xml.lineNumber() << xml.columnNumber();
                 m_map.clear();
                 return;
-                break;
             case QXmlStreamReader::EndDocument:
                 m_map.clear();
                 return;
+            default:
                 break;
             } // switch
         } // while

